scripts/check-size.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
 #
+
 # While Solana doesn't support resizing programs, the oracle binary needs to be smaller than 81760 bytes
+# (The available space for the oracle program on pythnet is 88429 and mainnet is 81760)
```

Comment:
i've bumped this up to the pythnet size so we can get this out for the audit. we can circle back and figure out how to split out mainnet/pythnet into separate builds.

(I think there's actually some code we can delete as well around the price account keys being signers and such, and that could help)